apps/backoffice-v2/src/common/hooks/useZodSearchParams/dateRangeSchema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+export const dateRangeSchema = z.object({
+  from: z
+    .string()
+    .optional()
+    .transform(val => (val ? new Date(val) : undefined)),
```

Comment:
We should start by adding the date input. Then the schema would match the input and output of the date input.

Reply:
I agree with you; and will implement that in upcoming task #2270 🙂